<map id="PyAttributeErrorObject" name="PyAttributeErrorObject">
<area shape="rect" id="node1" title=" " alt="" coords="308,5,476,32"/>
<area shape="rect" id="node2" href="$struct__object.html" title=" " alt="" coords="5,448,75,475"/>
<area shape="rect" id="node3" href="$struct__typeobject.html" title=" " alt="" coords="633,553,732,580"/>
<area shape="rect" id="node4" href="$structPyMappingMethods.html" title=" " alt="" coords="317,295,467,321"/>
<area shape="rect" id="node6" href="$structPyMethodDef.html" title=" " alt="" coords="337,359,447,385"/>
<area shape="rect" id="node7" href="$structPyBufferProcs.html" title=" " alt="" coords="336,424,448,451"/>
<area shape="rect" id="node8" href="$structPySequenceMethods.html" title=" " alt="" coords="313,525,471,552"/>
<area shape="rect" id="node9" href="$structPyGetSetDef.html" title=" " alt="" coords="339,579,445,605"/>
<area shape="rect" id="node10" href="$structPyNumberMethods.html" title=" " alt="" coords="319,753,465,780"/>
<area shape="rect" id="node11" href="$structPyAsyncMethods.html" title=" " alt="" coords="325,819,459,845"/>
<area shape="rect" id="node5" href="$structPyMemberDef.html" title=" " alt="" coords="335,968,449,995"/>
</map>
